packages/wrangler/src/__tests__/r2.test.ts

```diff
@@ -332,7 +332,7 @@ describe("r2", () => {
 					  "host": "api.cloudflare.com",
 					}
 				`);
-						response.once(
+						return response.once(
```

Comment:
ugh... ty for catching these, little disappointed the tests were passing without them returning the response properly.

Reply:
After I finishing converting the other tests, for sure.

Reply:
Likely something like this in these tests too https://github.com/cloudflare/wrangler2/pull/2377#discussion_r1045932416